src/resources/views/crud/buttons/quick.blade.php

```diff
@@ -37,3 +55,75 @@
         {{ $label }}
     </{{ $wrapper['element'] }}>
 @endif
+
+
+@if($ajax_enabled)
+{{-- Button Javascript --}}
+{{-- - used right away in AJAX operations (ex: List) --}}
+{{-- - pushed to the end of the page, after jQuery is loaded, for non-AJAX operations (ex: Show) --}}
+@push('after_scripts') @if (request()->ajax()) @endpush @endif
+@bassetBlock('backpack/crud/buttons/quick-button-'.app()->getLocale().'.js')
+<script>
+	if (typeof sendQuickRequest != 'function') {
+	  $("[data-button-type=quick]").unbind('click');
```

Comment:
This will unbind the clicks of all `quick` buttons, even the ones that are not ajax, won't that be a problem since the non-ajax buttons have no fallback to this function?

Reply:
OK, but why is it present in the [delete button](https://github.com/Laravel-Backpack/CRUD/blob/f2257fe6e0da458bd9d170d353b15ec66a2ad254/src/resources/views/crud/buttons/delete.blade.php#L15)? 

It was there, so I added it.

Reply:
Ok... But this `$wrapper['data-button-type'] = 'quick'`  is only attached when the button has the `ajax` attribute set. So it will not affect other non-ajax quick buttons.

Tell me your final words, I'll do it... I am just trying to understand why `unbind` is there on the delete button which made me copy it here. 😅

This discussion is just to learn and not repeat later. Thanks, Master

Reply:
Disregard then, in that case it's not affecting the other buttons like you said. 

I thought that was an attribute on ALL quick buttons, since the name is a generic "quick", if it would be `['data-button-type'] = 'quick-ajax'` I wouldn't have this doubt. 

Maybe worth to rename it, we don't have the "need" for adding `button-type` to non ajax **now**, but in the future we may need. And if that happens we will need to come up with a different key or making a breaking change. 

Let's start then with `= 'quick-ajax'` as the button type.

Reply:
Renaming `quick` to `quick-ajax`.